packages/botonic-react/src/components/whatsapp-button-list.tsx

```diff
@@ -1,11 +1,16 @@
 import { INPUT } from '@botonic/core'
 import React from 'react'
 
+import { truncateText } from '../util'
 import { renderComponent } from '../util/react'
 import { Message } from './message'
+import { WHATSAPP_MAX_BUTTON_CHARS } from './multichannel/multichannel-utils'
 
 // TODO: Add validation in component
 
+export const WHATSAPP_MAX_BUTTON_LIST_CHARS = 24
+export const WHATSAPP_MAX_BUTTON_LIST_DESCRIPTION_CHARS = 72
+export const WHATSAPP_MAX_BUTTON_LIST_ID_CHARS = 200
```

Comment:
This is the max of the payload? payload in flow builder is `payload|payloadParams` and payloadParms is JSON.stringify

Reply:
The payload of an option inside a WhatsApp Button List is 200 characters and the payload of a Whatsapp button is 256 characters (check this [documentation](https://developers.facebook.com/docs/whatsapp/on-premises/reference/messages#section-object))